tox.ini

```diff
@@ -127,12 +127,13 @@ commands =
     lint : python -m sphinx {[docs]sphinx_args_extra} -b linkcheck . {[docs]build_dir}/links
     all  : python -m sphinx {[docs]sphinx_args_extra} -b html . {[docs]build_dir}/html
 
-[testenv:package]
+[testenv:package{,-with-automation}]
 skip_install = True
 passenv = FORCE_COLOR
 deps =
     build==1.0.3
     twine==4.0.2
 commands =
-    python -m build --outdir dist/ .
+    python -m build . --outdir dist/
+    with-automation: python -m build automation/ --outdir dist/
```

Comment:
Actually - I take that back: the -with-automation target has been removed from the GitHub CI (it didn't show up in my earlier review because the CI workflow doesn't appear in the patch any more); but it doesn't appear to be used by the .github repos either. Am I missing something?

Reply:
I [introduced](https://github.com/beeware/.github/pull/69/files#diff-cda3ef2c3e868ac57017648b0c5053246f4d1e56164f94dec64b10975fc73283R22) the ability to pass additional tox factors to the `python-package-create` workflow with beeware/briefcase#1460. And passing unknown inputs to a variable causes CI to fail; so, I'll need to create another PR here that'll need to be merged with beeware/briefcase#1460.

Reply:
Sorry - is #1460 the right reference there?

Reply:
my bad - too many tabs open - I meant https://github.com/beeware/.github/pull/69